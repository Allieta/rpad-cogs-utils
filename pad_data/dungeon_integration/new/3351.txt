Gilles Legato Descended! - Millennium Castle's Demon Lord-Annihilation

stage 0

2577 - Ancient Flame Dragon, Ammonite @ level 15
Shell Seclusion
Reduce damage from all sources by 50% for 1 turns (Preemptive)
Wind-Up Full Throttle
Increase damage to 200% for the next 999 turns (When 1 enemy remains)
----- HP <= 100 -----
Three Hearts Clutch
18522 Damage - Spawn random 3 Heal orbs, Deal 130% damage (34% chance)
Seven Ties Tentacle
14248 Damage - Lock 7 random Fire, Water, Wood, Light, Dark orbs, Deal 100% damage (33% chance)
Nine Flames Breath
14248 Damage - Spawn random 9 Fire orbs, Deal 100% damage (33% chance)
----- HP <= 50 -----
----- Turn 1 -----
Three Hearts Clutch
18522 Damage - Spawn random 3 Heal orbs, Deal 130% damage (34% chance)
Seven Ties Tentacle
14248 Damage - Lock 7 random Fire, Water, Wood, Light, Dark orbs, Deal 100% damage (33% chance)
Nine Flames Breath
14248 Damage - Spawn random 9 Fire orbs, Deal 100% damage (33% chance)
----- Turn 2 -----
Shell Seclusion
Reduce damage from all sources by 50% for 1 turns
----- HP < 50 -----
----- Turn 1 -----
Incandescent Ocean
Change all orbs to Fire (1 time use)
----- Turn 2 -----
Three Hearts Clutch
18522 Damage - Spawn random 3 Heal orbs, Deal 130% damage (34% chance)
Seven Ties Tentacle
14248 Damage - Lock 7 random Fire, Water, Wood, Light, Dark orbs, Deal 100% damage (33% chance)
Nine Flames Breath
14248 Damage - Spawn random 9 Fire orbs, Deal 100% damage (33% chance)
----- Turn 3 -----
Shell Seclusion
Reduce damage from all sources by 50% for 1 turns
----- HP < 10 -----
Full Spiral
Player -100% HP

2580 - Ancient Ocean Dragon, Coelacanth @ level 15
Ability
Survive attacks with 1 HP when HP > 76%

stage 1

919 - Elemental of Harsh Flame, Shaitan @ level 12
I will devour you!
33543 Damage - Change 3rd row to Fire, Jammer orbs, Deal 130% damage (Preemptive)
----- Turn 1 -----
Flame Cage
Bind 3 random cards for 3 turns (1 time use)
----- Turn 2 -----
Dusky Malice
25802 Damage - Change all orbs to Jammer, Deal 100% damage
----- HP <= 100 -----
----- Execute repeatedly. Turn 3 -----
Dusky Malice
25802 Damage - Change all orbs to Jammer, Deal 100% damage
----- Loop to 1 after. Turn 4 -----
Burn Knuckle
38704 Damage - Deal 150% damage (2 hits, 75% each) (1 time use)
----- HP < 50 -----
----- Turn 2 -----
Flame Commandment
Bind active skills for 5 turns
----- Turn 3 -----
Dusky Malice
25802 Damage - Change all orbs to Jammer, Deal 100% damage

stage 2

2740 - Reincarnated Torrential Fenrir Knight, Kamui @ level 14
Fenrir Knight's Soul + Ice Axe
Bind active skills for 5 turns + Movetime -1s for 5 turns (Preemptive) (1 time use)
----- Turn 1 -----
Instant Strike
15826 Damage - Deal 50% damage (1 time use)
----- Turn 2 -----
Heavy Strike
31652 Damage - Deal 100% damage (2 hits, 50% each) (1 time use)
----- Turn 3 -----
Roaring Strike
47478 Damage - Deal 150% damage (3 hits, 50% each) (1 time use)
----- Turn 4 -----
Slicing Strike
63304 Damage - Deal 200% damage (4 hits, 50% each) (1 time use)
----- HP <= 100 -----
Destroying Strike
253216 Damage - Deal 800% damage (4 hits, 200% each)

stage 3

100910 - Alt. Black Baron @ level 10
Ability
Reduce damage from Healer, Attacker by 50%
Summon Poison Orbs!... + If you don't want to die
Poison skyfall +15% for 7 turns + Specific orbs change every 1.0s for 7 turns (Preemptive) (1 time use)
----- When afflicted by delay/poison -----
Resentment
Increase damage to 200% for the next 999 turns
----- HP <= 100 -----
Dark Slash
29928 Damage - Change Poison to Dark, Deal 290% damage

stage 4

4121 - Gilles Legato @ level 10
Overcoat of Immortality
Immune to damage from all sources for 999 turns (Preemptive)
Vampire Blade
16160 Damage - Change 2nd column to Dark orbs, Deal 160% damage (30% chance)
Blood Needle
18180 Damage - Deal 180% damage (2 hits, 90% each)
----- When 2 enemy remains -----
Dance!
Leader changes to random sub for 1 turns
----- HP <= 100 -----
Dark World
Blind all orbs on the board (60% chance)
Pain
Player -99% HP
----- When 1 enemy remains -----
Futile...
Enemy ally recover 100% hp

4123 - Red Stone Pillar @ level 10
Basic attacks (1)

4124 - Blue Stone Pillar @ level 10
Vile Cerulean Flame
Absorb combo <= 6 damage for 99 turns (Preemptive) (1 time use)
Revival Flame
Enemy ally recover 100% hp

