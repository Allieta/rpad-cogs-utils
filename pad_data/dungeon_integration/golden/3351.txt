Gilles Legato Descended! - Millennium Castle's Demon Lord-Annihilation

stage 0

2577 - Ancient Flame Dragon, Ammonite @ level 15

Shell Seclusion
Reduce damage from all sources by 50% for 1 turns
----- HP <= 50 -----
HP <= 50
Incandescent Ocean
Change all orbs to Fire
----- HP <= 10 -----
HP <= 10
Full Spiral
Player -100% HP

2580 - Ancient Ocean Dragon, Coelacanth @ level 15

Coelacanth's Temper
Survive attacks with 1 HP when HP > 76%

stage 1

919 - Elemental of Harsh Flame, Shaitan @ level 12

I will devour you!
Change 3rd row to Fire, Jammer orbs, Deal 130% damage
----- Turn 1 -----
Turn 1
Flame Cage
Bind 3 random cards for 3 turns
----- Turn 2 -----
Turn 2
Burn Knuckle
Deal 150% damage (2 hits, 75% each)
----- HP <= 100 -----
HP <= 100
Dusky Malice
Change all orbs to Jammer, Deal 100% damage
----- HP <= 49 -----
HP <= 49
Flame Commandment
Bind active skills for 5 turns

stage 2

2740 - Reincarnated Torrential Fenrir Knight, Kamui @ level 14

Fenrir Knight's Soul + Ice Axe
Bind active skills for 5 turns + Movetime -1s for 5 turns
----- Turn 1 -----
Turn 1
Instant Strike
Deal 50% damage
----- Turn 2 -----
Turn 2
Heavy Strike
Deal 100% damage (2 hits, 50% each)
----- Turn 3 -----
Turn 3
Roaring Strike
Deal 150% damage (3 hits, 50% each)
----- Turn 4 -----
Turn 4
Slicing Strike
Deal 200% damage (4 hits, 50% each)
----- HP <= 100 -----
HP <= 100
Destroying Strike
Deal 800% damage (4 hits, 200% each)

stage 3

100910 - Alt. Black Baron @ level 10

Healer & Attacker Type Halved
Reduce damage from Healer, Attacker by 50%
Summon Poison Orbs!... + If you don't want to die
Poison skyfall +15% for 7 turns + Specific orbs change every 1.0s for 7 turns
----- Turn 1 -----
Turn 1
Resentment
Increase damage to 200% for the next 999 turns
----- HP <= 100 -----
HP <= 100
Dark Slash
Change Poison to Dark, Deal 290% damage

stage 4

4121 - Gilles Legato @ level 10

Overcoat of Immortality
Immune to damage from all sources for 999 turns
----- HP <= 100 -----
HP <= 100
Vampire Blade
Change 2nd column to Dark orbs, Deal 160% damage
Blood Needle
Deal 180% damage (2 hits, 90% each)

4124 - Blue Stone Pillar @ level 10

Vile Cerulean Flame
Absorb combo <= 6 damage for 99 turns
----- HP <= 100 -----
HP <= 100
Revival Flame
Enemy ally recover 100% hp

4123 - Red Stone Pillar @ level 10

This monster will only use basic attacks (or computing enemy skills failed)
